### PR TITLE
Improve PDF render experience

### PR DESCRIPTION
**Pull request by mdbenjam**
This PR does three things to improve PDF rendering:
1) Upgrade PDFJS to the latest version under major version 1.
1) Change the overscan behavior to work in both directions, not just the direction the user is scrolling.
1) Move calculating page dimensions to the initial load of the document. Even for a 2500 page document it only takes ~2 seconds, but if done lazily while scrolling it slows down the application.

These three things seem to improve rendering performance, it's hard to measure, so I'll be curious what the reviewer thinks.

**Test Plan**
- [ ] Play around with PDFs locally.

### CODE REVIEW

File: client/app/reader/PdfFile.jsx [hunk 1]
```diff
@@ -116,8 +116,6 @@ export class PdfFile extends React.PureComponent {
 
     return <div key={pageIndex} style={style}>
       <PdfPage
-        scrollTop={this.props.scrollTop}
```
Comment (mdbenjam) [inline client/app/reader/PdfFile.jsx]: Unused, and PdfPage shouldn't re-render based on these.

File: client/app/reader/PdfFile.jsx [hunk 2]
```diff
@@ -402,6 +400,11 @@ export class PdfFile extends React.PureComponent {
     </div>;
   }
 
+  overscanIndicesGetter = ({ cellCount, overscanCellsCount, startIndex, stopIndex }) => ({
```
Comment (mdbenjam) [inline client/app/reader/PdfFile.jsx]: Now we draw pages in both directions instead of just the scrolling direction.

File: client/package.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
     "moment": "^2.18.1",
     "moment-timezone": "^0.5.13",
     "pdf-annotate.js": "^1.0.0",
-    "pdfjs-dist": "1.6.414",
+    "pdfjs-dist": "1.10.97",
```
Comment (mdbenjam) [inline client/package.json]: We were massively behind...

File: client/app/reader/PdfFile.jsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```
Comment (mdbenjam) [inline client/app/reader/PdfFile.jsx]: This pains me, but I think it's our best choice for right now. We can try to tighten this file up more later.

File: client/app/reader/PdfPage.jsx [hunk 5]
```diff
@@ -226,24 +229,13 @@ export class PdfPage extends React.PureComponent {
...
         catch(() => {
           // We might need to do something else here.
         });
     }
   }
 
-  getDimensions = (page) => {
-    const viewport = page.getViewport(PAGE_DIMENSION_SCALE);
-
-    this.props.setPageDimensions(
```
Comment (dannysellers) [inline client/app/reader/PdfPage.jsx]: It looks like `setPageDimensions` is now unused in `PdfPage` (it's still imported)

**Pull request merged**